src/rgw/rgw_bucket.cc

```diff
@@ -2529,6 +2544,17 @@ int RGWMetadataHandlerPut_BucketInstance::put_check()
 
   bool exists = (!!orig_obj);
 
+  cct = bihandler->svc.bucket->ctx();
```

Comment:
@cbodley I have initialized cct here. Not sure if its a good idea. Also, I don't see cct being used in any other methods of this class, except for logging.

Reply:
the `RGWMetadataHandlerPut_BucketInstance` constructor already takes a CephContext pointer, it just doesn't use that to initialize the member variable. can you fix the constructor instead?